src/viam/__init__.py

```diff
@@ -1,71 +1,73 @@
-import sys
-from importlib.metadata import PackageNotFoundError, version
-from uuid import uuid4
-
-from viam.gen.common.v1.common_pb2 import ResourceName as _ResourceName

```

Comment:
I don't understand what's going on here. I know that `update_api_version.py` asked for the file to be re-written, but it looks the same. I also checked on diffchecker, and it was confirmed that nothing changed.